toplevel/coqtop.ml

```diff
@@ -58,6 +58,7 @@ type ('a,'b) custom_toplevel =
 (** Main init routine *)
 let init_toplevel { parse_extra; init_extra; usage; initial_args } =
   Coqinit.init_ocaml ();
+  Vernacextend.static_linking_done ();
```

Comment:
IMO this should be called by `init_ocaml`, not by all toplevels.

Reply:
init_ocaml looks pretty generic so far
~~~ocaml
let init_ocaml () =
  CProfile.init_profile ();
  init_gc ();
  Sys.catch_break false (* Ctrl-C is fatal during the initialisation *)
~~~
so it feels like it shouldn't set flags in a module like Vernacextend

Maybe that means we should move the flag?

Reply:
init_runtime looks reasonable